eo.json

```diff
@@ -1212,7 +1212,7 @@
 
         "Tutorial.Welcome.StartTutorialHeader" : "Komenci Lernilon",
         "Tutorial.Welcome.StartTutorialDescription" : "Ni translokos vin al baza lernila mondo por helpi vin komenci kun Neos. Vi povas elekti aŭ havi la lernil-mondon publika, permesante al Neos-komunumo aliĝi kaj helpi vin, aŭ lerni memstare.",
-        "Tutorial.Welcome.StartTutorialPublic" : "Mi volas, ke homoj aliĝi kaj helpi min",
+        "Tutorial.Welcome.StartTutorialPublic" : "Mi volas, ke homoj aliĝu kaj helpi min",
```

Comment:
Mankis -u en la dua verbo: "...aliĝu kaj _helpu_ min"

Reply:
Ah! 😅 Thanks for catching that!